packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -466,7 +466,16 @@ const helperBehavior = {
 
         // The rest of this method will not happen until after a quote is received.
         // This may not happen until much later, when the market changes.
-        // eslint-disable-next-line no-await-in-loop
+        //
+        // This nested await is safe because "terminal-control-flow"
```

Comment:
This should refer to the `for/await/generator/while pattern` that is used in several places for vaults.